docs/graph-algorithms/breadth-first-search.mdx

```diff
@@ -0,0 +1,341 @@
+---
+title: Breadth First Search
+slug: /graph-algorithms/breadth-first-search
+---
+## Introduction
+
+Breadth-first search (BFS) is an algorithm for traversing or searching tree or graph data structures.It is similar to level-order traversal in trees. It starts at the root node and explores all of the neighbor nodes at the present node before moving on to the nodes at the next depth level.BFS uses queue data structure for traversing the nodes.
```

Comment:
give space after the full stop in the first line.

write it as  "a queue" in the last line

Reply:
you can write it as : 

Breadth-first search (BFS) is an algorithm to traverse/search the data structure tree or a graph.

Reply:
I have run the theory through Grammarly once and done the changes that you suggested and committed them, do check once